src/clarity-angular/select/select.html

```diff
@@ -0,0 +1,12 @@
+<ng-content select="[clrSelectInput]"></ng-content>
+<button type="button" class="btn-icon btn btn-link open-trigger" (click)="onMenuTriggerClick($event)">
+    <clr-icon aria-hidden="true" shape="angle down"></clr-icon>
+</button>
+<clr-select-menu *clrIfOpen style="width: 300px;">
```

Comment:
I didn't realize this last time, but you don't want to remove the menu from the DOM for all cases.
Simple selects without `*clrIfOpen` on the app side should have all options available in the DOM for screenreaders to have access to.